try {
    hello();
} catch (error) {
    console.log(error);
}

    //PERSONALIZABLE
try {
    hello();
} catch{
    console.log("\n Este es un error personalizable");
}